### PR TITLE
[11.0] [MIG] base_rest, base_rest_demo: Migration to 11.0

### PR DESCRIPTION
**Pull request by lmignon**

### CODE REVIEW

File: base_rest/http.py [hunk 1]
```diff
@@ -43,11 +43,18 @@ def wrapJsonException(exception):
     get_original_headers = exception.get_headers
 
     def get_body(environ=None):
-        return JSONEncoder().encode({
+        res = {
             'code': exception.code,
             'name': escape(exception.name),
             'description': exception.get_description(environ)
+            }
+        if _logger.isEnabledFor(logging.DEBUG):
```
Comment (sbidoul) [inline base_rest/http.py]: This sounds dangerous. Enabling `--log-level=debug` globally will have the (surprising IMO) side effect of enabling stack traces in base_rest (which is often considered a security issue).

I'd prefer an option in the config file, that we could then manage with `server_environment`.

Comment (lmignon) [reply base_rest/http.py]: @sbidoul Thank you for this insightful comment. I'll change the code to use an option into the config file.

Comment (lmignon) [reply base_rest/http.py]: @sbidoul What do you think about https://github.com/OCA/rest-framework/pull/5/commits/317dcb4a91ae049af6c6778c6e6ef54222f31f77

Comment (sbidoul) [reply base_rest/http.py]: I made a couple of comments on the commit.

File: .travis.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ sudo: false
 cache: pip
 
 python:
-  - "2.7.13"
+  - "3.6"
```
Comment (sbidoul) [inline .travis.yml]: For 11.0 we need to test on 3.5 which is the minimum version supported by Odoo.

File: setup/base_rest_demo/odoo/addons/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+__import__('pkg_resources').declare_namespace(__name__)
```
Comment (sbidoul) [inline setup/base_rest_demo/odoo/addons/__init__.py]: no init.py for 11

**Review by sbidoul — APPROVED** [review]
A couple of minor changes, otherwise +1

**Comment by lmignon** [comment]
Thank you @sbidoul  Changes done

**Comment by sebastienbeau** [comment]
Ok for me :+1:

**Review by sebastienbeau — APPROVED** [review]

**Pull request merged**